features/step_definitions/integration_steps.rb

```diff
@@ -266,7 +266,7 @@ def non_dc_office_location
                             can_complete_resident_application: true, can_add_sep: true, can_view_username_and_email: true, can_view_application_types: true,
                             view_personal_info_page: true, can_access_outstanding_verification_sub_tab: true, can_access_identity_verification_sub_tab: true,
                             can_access_accept_reject_paper_application_documents: true, can_delete_identity_application_documents: true,
-                            can_access_accept_reject_identity_documents: true)
+                            can_access_accept_reject_identity_documents: true, can_edit_aptc: true)
```

Comment:
This was just for ME right?

Even though this is a cucumber we should probably use `tap` to add it in conditionally if its configured.

Reply:
No, it is for both ME and DC. Added a new attribute in the permission model, earlier we were using the can_update_ssn attribute for various features to turn permission level on or off now `Edit APTC / CSR` using its own level of permission attribute with can_edit_aptc.